pkgs/development/r-modules/default.nix

```diff
@@ -309,6 +309,7 @@ let
   };
 
   packagesWithNativeBuildInputs = {
+    asciicast = [ pkgs.lzma.dev pkgs.bzip2.dev pkgs.zlib.dev pkgs.icu.dev ];
```

Comment:
We should be more careful with these dependencies: all of these are not native build inputs but build inputs. The distinction is a bit moot with R but let's try to start cleaning these up.

Reply:
Ok, I thought that these were only needed to compile the package itself, so they’re needed at runtime as well? Where do you see that?

Reply:
> Ok, I thought that these were only needed to compile the package itself, so they’re needed at runtime as well? Where do you see that?

Dynamic libraries are needed at runtime, but it's better to think about cross-compiling and the build/target hosts rather than compile/runtime. Native build inputs are needed on the build host, for the build host architecture: this is stuff like pkg-config which the build host needs to run to configure the build, the (cross) compiler, etc. The remaining build inputs are required for the target host.